src/components/confirm/confirm.test.js

```diff
@@ -0,0 +1,388 @@
+import * as React from "react";
+import Confirm from "./confirm.component";
+import Button from "../button/button.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import {
+  dialogPreview,
+  dialogSubtitle,
+  confirmButton,
+  cancelButton,
+} from "../../../cypress/locators/confirm";
+import { getDataElementByValue, icon } from "../../../cypress/locators/index";
+import { positionOfElement, keyCode } from "../../../cypress/support/helper";
+
+const specialCharacters = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const heights = [
+  ["0", "height0"],
```

Comment:
If I pass a number into the height prop it seems to ignore it and leave height as the default. I'm now passing a string in there and using a number for the verification, which works.

Reply:
This is because the Dialog component is treating `0` as a falsy value and not applying it. If you pass `"0"` it will work

https://github.com/Sage/carbon/blob/a17b967ec74437a42ab156e3b4cc63fd4bdbbe7d/src/components/dialog/dialog.component.js#L154